legend-engine-xt-documentStore-executionPlan-connection/src/main/java/org/finos/legend/engine/plan/execution/stores/nonrelational/AlloyMongoServer.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.plan.execution.stores.nonrelational;
+
+import de.bwaldvogel.mongo.MongoServer;
+import de.bwaldvogel.mongo.backend.memory.MemoryBackend;
+import org.slf4j.Logger;
+
+public class AlloyMongoServer
+{
+    private static final Logger LOGGER = org.slf4j.LoggerFactory.getLogger(AlloyMongoServer.class);
+    protected static final String DEFAULT_MONGO_HOSTNAME = "localhost";
+    protected static final int DEFAULT_MONGO_PORT = 27017;
+
+    public static MongoServer startServer(int port)
+    {
+        LOGGER.debug("Starting setup of connection for local Mongo database server on port: " + port + " and address: " + DEFAULT_MONGO_HOSTNAME);
+        MongoServer mongoServer = new MongoServer(new MemoryBackend());
+        mongoServer.bind(DEFAULT_MONGO_HOSTNAME, port);
+        System.out.println("Completed setup for local Mongo database server on port: " + port + " and address: " + DEFAULT_MONGO_HOSTNAME);
```

Comment:
any specific reason for both logger & system.out?

Reply:
Missed this, using only LOGGER now